gradle.properties

```diff
@@ -1 +1 @@
-version=14.0.0
+version=14.0.1
```

Comment:
It is a breaking change so please update major version

Reply:
Changed to 15.0.0 in 85c0f5010899a1f6ec42a2150971f30553b7d8d6.